components/brave_wallet_ui/common/slices/api.slice.ts

```diff
@@ -1273,14 +1297,14 @@ export function createWalletApi () {
 
             return {
               data: registryArray.reduce((acc, curr) => {
-                for (const [address, chainIds] of Object.entries(
+                for (const [uniqueKey, chainIds] of Object.entries(
                   curr
                 )) {
-                  if (!acc.hasOwnProperty(address)) {
-                    acc[address] = chainIds
+                  if (!acc.hasOwnProperty(uniqueKey)) {
+                    acc[uniqueKey] = chainIds
                   } else {
-                    acc[address] = {
-                      ...acc[address],
+                    acc[uniqueKey] = {
+                      ...acc[uniqueKey],
```

Comment:
use `entityIdFromAccountId` to prevent this from breaking when/if changes to the account id selection logic are made


`entityIdFromAccountId` currently uses `address` as the default value